internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/PartialUpdateTest.java

```diff
@@ -491,6 +492,73 @@ public void testPartialUpdateOnBatchPushedKeys(CompressionStrategy compressionSt
     }
   }
 
+  @Test(timeOut = TEST_TIMEOUT_MS)
+  public void testPartialUpdateOnBatchPushedChunkKeys() throws IOException {
```

Comment:
Does this repro the issue, in the absence of the main code fix?

Reply:
Yes